### ISSUE TITLE
Browserify error: Cannot find module 'videojs'

### ISSUE THREAD

**Issue by alsar** [collab-project/videojs-record#27]
There seems to be an typo:  require('videojs'). When i change it to require('video.js') it works.

```Browserify Error { [Error: Cannot find module 'videojs' from '/vagrant/node_modules/videojs-record/src/js']```

```js
(function (root, factory)
{
    if (typeof define === 'function' && define.amd)
    {
        // AMD. Register as an anonymous module.
        define(['videojs'], factory);
    }
    else if (typeof module === 'object' && module.exports)
    {
        // Node. Does not work with strict CommonJS, but
        // only CommonJS-like environments that support module.exports,
        // like Node.
        module.exports = factory(require('videojs'));
    }
    else
    {
        // Browser globals (root is window)
        root.returnExports = factory(root.videojs);
    }
```

**Comment by thijstriemstra** [comment]
@alsar can you provide an example to reproduce this issue, I'm not familar with Browserify.

**Comment by alsar** [comment]
Like in this plugin, that also uses videojs:
https://github.com/eXon/videojs-youtube/blob/master/src/Youtube.js#L29

**Comment by thijstriemstra** [comment]
Thanks! Closed with #28
